diff -up ./misc/macros.jpackage.sav ./misc/macros.jpackage
--- ./misc/macros.jpackage.sav	2008-01-04 06:21:42.000000000 -0500
+++ ./misc/macros.jpackage	2008-07-08 14:41:49.000000000 -0400
@@ -88,11 +88,14 @@
 #
 %_javadocdir     %{_datadir}/javadoc
 
+# Directory for maven poms
+%_mavenpomdir %{_datadir}/maven2/poms
+
 #
 # Directory for maven depmaps
 #
-%_mavendepmapdir /etc/maven
-%_mavendepmapfragdir /etc/maven/fragments
+%_mavendepmapdir %{_sysconfdir}/maven
+%_mavendepmapfragdir %{_sysconfdir}/maven/fragments
 
 #
 # Current default JVM home.
